_includes/team-signup.html

```diff
@@ -0,0 +1,46 @@
+<form action="//studentrobotics.us12.list-manage.com/subscribe/post?u=1af17c2a3426de258620aae46&amp;id=dd7d04139d"
+      method="post"
+      id="mc-embedded-subscribe-form"
+      name="mc-embedded-subscribe-form"
+      class="form form-left show-labels validate"
+      target="_blank"
+      novalidate>
+    <div id="mc_embed_signup_scroll">
+      <div class="mc-field-group">
+        <label for="mce-EMAIL">Email Address</label>
+        <input type="email" value="" name="EMAIL" class="required email full-width" id="mce-EMAIL">
+      </div>
+      <div class="mc-field-group">
+        <label for="mce-FNAME">Your First Name</label>
+        <input type="text" value="" name="FNAME" class="" id="mce-FNAME" size="24">
+      </div>
+      <div class="mc-field-group">
+        <label for="mce-LNAME">Your Last Name</label>
+        <input type="text" value="" name="LNAME" class="" id="mce-LNAME" size="24">
+      </div>
+      <div class="mc-field-group">
+        <label for="mce-MMERGE4">Organisation</label>
+        <p class="explanation">If your team represents a school, or another organisation, enter its name here.</p>
+        <input type="text" value="" name="MMERGE4" class="full-width" id="mce-MMERGE4">
+      </div>
+      <div id="mce-responses" class="clear">
+        <div class="response" id="mce-error-response" style="display:none"></div>
+        <div class="response" id="mce-success-response" style="display:none"></div>
+      </div>
+      <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_1af17c2a3426de258620aae46_dd7d04139d" tabindex="-1" value=""></div>
+      <div class="clear">
+        <input class="button button-primary" type="submit" value="Register your interest" name="subscribe" id="mc-embedded-subscribe">
+        <span>or</span>
+        <a class="button" href="mailto:{{ site.emails.teams }}" target="_blank">Speak to someone</a>
+      </div>
+    </div>
+    <div id="mergeRow-gdpr" class="mergeRow gdpr-mergeRow content__gdprBlock mc-field-group">
+        <div class="content__gdpr">
+            <p>Student Robotics will use the information you provide on this form to contact you around leading a team at Student Robotics. We will contact you via email.</p>
+            <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="gdpr[24613]" tabindex="-1" value="Y"></div>
+        </div>
+        <div class="content__gdprLegal">
+            <p>We use MailChimp as our contacting platform. By clicking below to sign up, you acknowledge that your information will be transferred to MailChimp for processing. Learn more about MailChimp's privacy practices <a href="https://mailchimp.com/legal/" target="_blank">here</a>.</p>
```

Comment:
Should this also mention that we use Google too?

Reply:
We don't process mailing list contact using Google unless they sign up for a competition, at which point we state we use Google for this.

So I _think_ under that, we're at least telling the truth!